indico/modules/events/abstracts/blueprint.py

```diff
@@ -50,6 +50,7 @@
 _bp.add_url_rule('/manage/abstracts/open', 'open_abstracts_call', RHOpenCFA, methods=('POST',))
 _bp.add_url_rule('/manage/abstracts/close', 'close_abstracts_call', RHCloseCFA, methods=('POST',))
 _bp.add_url_rule('/manage/abstracts/boa', 'manage_boa', RHManageBOA, methods=('GET', 'POST'))
+_bp.add_url_rule('/book-of-abstracts.pdf', 'export_boa', RHExportBOA)
```

Comment:
Move this into the "Display pages" section. It's not management. `RHExportBOA` shouldn't be in management.py either. You could move it to display.py or add a new `boa.py` and move both this RH and RHManageBOA in there.